specification/support/resource-manager/readme.md

```diff
@@ -26,18 +26,44 @@ These are the global settings for the Support API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2020-04
+tag: package-preview-2022-09
 ```
 
 
+### Tag: package-preview-2022-09
+
+These settings apply only when `--tag=package-preview-2022-09` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2022-09'
+input-file:
+  - Microsoft.Support/preview/2022-09-01-preview/support.json
+suppressions:
+  - code: RPC-Put-V1-01
+    from: Microsoft.Support/preview/2022-09-01-preview/support.json
+    reason: "Rule: The path for put operation must be under a subscription and resource group. Justification: Suppressing this rule since path for support ticket resource doesn't contain resource group"
+  - code: RPC-Uri-V1-05
+    from: Microsoft.Support/preview/2022-09-01-preview/support.json
+    reason: "Rule: The resource name parameter supportTicketName should be defined with a pattern restriction. Justification: We have similar implementation for previous versions, this would be a breaking change, hence suppressing it"
+  - code: RPC-Put-V1-11
+    from: Microsoft.Support/preview/2022-09-01-preview/support.json
+    reason: "Rule: Any Put MUST contain 200 and 201 return codes. Justification: We have similar implementation for previous versions, this would be a breaking change, hence suppressing it"
+  - code: RPC-Patch-V1-03
```

Comment:
Having similar implementation in previous version is not a good rationale. Can u specify a valid reason for the design choice.

Reply:
As part of this version, we are doing some minor model updates and added a new API fileworkspace. There were multiple meetings before coming up with design for fileworkspace and implemented accordingly and all other apis are same as previous versions. A detailed explanation was provided to ARM reviewer for the new additions and got an approval for private preview pr.